internal/common/utils/Forces.go

```diff
@@ -0,0 +1,7 @@
+package forces
+
+type Forces struct {
+	pedal   float64
+	brake   float64
+	turning float64
+}
```

Comment:
I really like this. I'd consider having that `commonTypes.go` file though, where you can put colour, force and coordinate/position